### PR TITLE
Add classifier experiments notebook

### PR DESCRIPTION
**Pull request by tm-jace-peralta**
This PR includes a classifier experiment notebook that (1) categorizes the output of the regression model into 5 income groups (A-richest to E-poorest), and (2) creates matrix-based metrics to evaluate the model's performance when its output is grouped into those income categories.

### CODE REVIEW

File: notebooks/2023-02-15-dhs-cross-country-classifier-experiments/2023-02-15_crosscountry_normalized_classifier.ipynb [hunk 1]
```diff
@@ -0,0 +1,1507 @@
+{
```
Comment (alronlam) [inline notebooks/2023-02-15-dhs-cross-country-classifier-experiments/2023-02-15_crosscountry_normalized_classifier.ipynb:1]: <p>@jace would recommend let's not attach interpretations like this. For the dev sector, wealth means a lot of things (income, assets, etc). Even then, our data is asset-based; not income. </p><p> </p><p>Maybe rephrase to something like A = "wealthiest" -&gt; E = "poorest" </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thinkingmachines/unicef-ai4d-poverty-mapping/pull/151/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

**Review by alronlam — APPROVED** [review]
Approving now, but please address the comment I left. Thanks! :)

**Comment by tm-jace-peralta** [comment]
> Approving now, but please address the comment I left. Thanks! :)

done, thanks @alronlam!
